amt/api/routes/project.py

```diff
@@ -35,3 +37,123 @@ async def get_root(
         raise NotFound()
 
     return templates.TemplateResponse(request, "pages/index.html.j2", context)
+
+
+# !!!
+# Implementation of this endpoint is for now independent of the project ID, meaning
+# that the same system card is rendered for all project ID's. This is due to the fact
+# that the logical process flow of a system card is not complete.
+# !!!
+@router.get("/{project_id}/system_card")
+async def get_system_card(
+    request: Request,
+    project_id: int,
+    projects_service: Annotated[ProjectsService, Depends(ProjectsService)],
+    file_system_storage_service: Annotated[FileSystemStorageService, Depends(FileSystemStorageService)],
```

Comment:
I don't think this should be a dependency this way as you can not initialize it properly and I'm not sure what object you get (is it a new instance or reused). I think this line should go, and in your code you should use:
system_card: Any = StorageFactory.init(storage_type="file", location=path.parent, filename=path.name).read()
as we did in the check_state.py code (line 49).